docs/Documentation/Scripting/Building-Blocks/Conditions-List.md

```diff
@@ -512,12 +512,21 @@ denizenVariableThis: "variable %ph.denizen_<player.has_flag[flag_name]>% true fo
 
 ## Weather: `weather`
 
+**static**
+
 There must be a specific weather for this condition to return true. There are three possible options: sun, rain and storm. Note that `/toggledownfall` does not change the weather, it just does what the name suggests: toggles downfall. The rain toggled off will still be considered as rain! Use `/weather clear` instead.
 
-!!! example
-    ```YAML
-    weather sun
-    ```
+| Parameter | Syntax     | Default Value          | Explanation                         |
+|-----------|------------|------------------------|-------------------------------------|
+| _weather_ | Keyword    | :octicons-x-circle-16: | The weather to check for.           |
+| _world_   | world:name | player location        | The world to check for the weather. |
+
+
+```YAML title="Example"
+isSunny: "weather sun"
+weatherInPlayerWorld: "weather rain world:%player.homeWorld%"
```

Comment:
That variable is not existent in BetonQuest. You could use a non-existent PAPI variable instead.